src/lib.rs

```diff
@@ -857,10 +857,33 @@ impl BumpFeeTxBuilder {
     }
 }
 
-fn generate_mnemonic(word_count: WordCount) -> Result<String, BdkError> {
-    let mnemonic: GeneratedKey<_, BareCtx> =
-        Mnemonic::generate((word_count, Language::English)).unwrap();
-    Ok(mnemonic.to_string())
+struct Mnemonic {
+    internal: BdkMnemonic,
+}
+
+impl Mnemonic {
```

Comment:
We're starting to enforce that documentation be written for new PRs. In general it's not too hard because you can more or less use what is in the Rust docs if the constructs match.